nx/include/switch/services/fs.h

```diff
@@ -88,6 +88,13 @@ typedef struct {
     u8 unused[0x190];               ///< Uninitialized.
 } FsSaveDataExtraData;
 
+/// FsSaveMeta
+typedef struct {
+    u32 meta_file_size;
+    u8 meta_index;
+    u8 unk[0x0B];
+} FsSaveMeta;
```

Comment:
This is "nn::fs::SaveDataMetaInfo", rename to FsSaveDataMetaInfo.

Rename "unk" to "reserved", since that's what the field is.

Reply:
Also, consider renaming "meta_index" to "meta_type", since this is nn::fs::SaveDataMetaType (enum).

Enum values are:

```
None             = 0,
Thumbnail        = 1,
ExtensionContext = 2,
```

Reply:
Honestly, rename the fields as follows:

```
meta_file_size -> size
meta_index -> type
unk -> reserved
```